Build-exe-and-msi.py

```diff
@@ -0,0 +1,307 @@
+                'dataclasses',
+                'shutil',  # Included for plugins
+                'timeout_session',
+                'zipfile',  # Included for plugins
+            ],
+            'excludes': [
+                'distutils',
+                '_markerlib',
+                'optparse',
+                'PIL',
+                'simplejson',
+                'unittest'
+            ],
+        }
+    }
+
+    DATA_FILES = [
+        ('', [
+            '.gitversion',  # Contains git short hash
+            'WinSparkle.dll',
+            'WinSparkle.pdb',  # For debugging - don't include in package
+            'EUROCAPS.TTF',
+            'ChangeLog.md',
+            'snd_good.wav',
+            'snd_bad.wav',
+            'modules.p',
+            'ships.p',
+            f'{appname}.VisualElementsManifest.xml',
+            f'{appname}.ico',
+            'EDMarketConnector - TRACE.bat',
+            'EDMarketConnector - localserver-auth.bat',
+            'EDMarketConnector - reset-ui.bat',
+        ]),
+        ('L10n', [join('L10n', x) for x in os.listdir('L10n') if x.endswith('.strings')]),
+        ('FDevIDs', [
+            join('FDevIDs', 'commodity.csv'),
+            join('FDevIDs', 'rare_commodity.csv'),
+        ]),
+        ('plugins', PLUGINS),
+    ]
+
+else:
+    raise AssertionError('Unsupported platform')
+###########################################################################
+
+###########################################################################
+# Use py2exe's `freeze()` to produce the executables.
+###########################################################################
+freeze(
+    version_info={
+        'description': 'Downloads commodity market and other station data from the game Elite Dangerous for use with'
+                       ' all popular online and offline trading tools.',
+        'company_name': 'EDCD',  # Used by WinSparkle
+        'product_name': appname,  # Used by WinSparkle
+        'version': base_appversion,
+        'product_version': appversion_str,
+        'copyright': copyright,
+        'language': 'English (United States)',
+    },
+    windows=[
+        {
+            'dest_base': appname,
+            'script': APP,
+            'icon_resources': [(0, f'{appname}.ico')],
+            'other_resources': [(24, 1, open(f'{appname}.manifest').read())],
+        }
+    ],
+    console=[
+        {
+            'dest_base': appcmdname,
+            'script': APPCMD,
+            'other_resources': [(24, 1, open(f'{appcmdname}.manifest').read())],
+        }
+    ],
+    data_files=DATA_FILES,
+    options=OPTIONS,
+)
+###########################################################################
+
+###########################################################################
+# Build installer(s)
+###########################################################################
+package_filename = None
+# Ensure this fails on non-win32
+if sys.platform == 'win32':
```

Comment:
```suggestion
if sys.platform != 'win32':
    raise AssertionError(...)
```

Or `assert platform == 'win32'`?

Reply:
I've removed all the `win32` tests except the first one now.  I was too much in "minimal changes" mindset the other day.